### PR TITLE
Proof formats - edits

### PR DESCRIPTION
**Pull request by brentzundel**
Combined `Proofs` and `Cryptography` into a single section called `Proof Formats`
Edited for clarity, line length, spacing, and to add links to terminology.
Removed a bunch of whitespace (and added a bit).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -722,26 +702,49 @@ <h3>Human Readability</h3>
...
+specification</a> does not normatively require any particular digital proof or
+signature format. While the data model is the canonical representation of a
+<a>verifiable credential</a> or <a>verifiable presentation</a>, the proving
+mechanisms for these are often tied to the syntax used in the transmission of
+the document between parties. As such, each proofing mechanism has to specify
+whether the validation of the proof is calculated against the state of the
+document as transmitted, against the transformed data model, or against another
+form. At the time of publication, at least two proof formats are being actively
+utilized by implementers and the Working Group felt that documenting what these
+proof formats are and how they are being used would be beneficial to
```
Comment (TallTed) [inline index.html]: `beneficial to`
-> `beneficial to other`

File: index.html [hunk 2]
```diff
@@ -722,26 +702,49 @@ <h3>Human Readability</h3>
...
+proof format agnostic. <a href="https://w3c.github.io/vc-data-model/">The
+specification</a> does not normatively require any particular digital proof or
+signature format. While the data model is the canonical representation of a
+<a>verifiable credential</a> or <a>verifiable presentation</a>, the proving
+mechanisms for these are often tied to the syntax used in the transmission of
+the document between parties. As such, each proofing mechanism has to specify
+whether the validation of the proof is calculated against the state of the
+document as transmitted, against the transformed data model, or against another
+form. At the time of publication, at least two proof formats are being actively
+utilized by implementers and the Working Group felt that documenting what these
```
Comment (TallTed) [inline index.html]: `implementers and`
-> `implementers, and`

File: index.html [hunk 3]
```diff
@@ -1395,33 +1397,33 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
 In order to encode a <a>verifiable credential</a> or a
 <a>verifiable presentation</a> in a JWT, an extra set of steps
 are required to convert the data to and from the JWT format. No such extra
 converstion step are required for <a>verifiable credentials</a> and
 <a>verifiable presentations</a> protected by LD-Proofs.
-          </dd>
-          <dt id="pf19b">
+            </dd>
+            <dt id="pf19b">
 PF19b: Canonicalization requires more than base-64 encoding.
```
Comment (TallTed) [inline index.html]: _**EDITED** -- initial rewording to `does not require more than` was clumsy; new `requires only` is much better._

PF19b is currently not worded as a Benefit, but as a Cost -- and breaks from all the other items in the list.  It should be reworded --

`PF19b: Canonicalization requires more than base-64 encoding.`
-> `PF19b: Canonicalization requires only base-64 encoding.`

-- and the entry, checks, and x'es in the summary table (lines 1180-1184, in the current PR) should likewise be reversed, becoming --

```
<a href="#pf19b">PF19b.</a> Canonicalization requires only base-64 encoding.
                </td>
                <td class="missing">✖</td>
                <td class="missing">✖</td>
                <td class="supported">✓</td>
```

File: index.html [hunk 4]
```diff
@@ -1395,33 +1397,33 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 The JWT format utilizes a simple base-64 encoding format to generate the
 cryptographic hash of the data. The encoding format for LD-Proofs require
```
Comment (TallTed) [inline index.html]: `format for LD-Proofs require`
-> `format for LD-Proofs requires`
(add `s` to `require`)

**Comment by TallTed** [comment]
I will likely have more on this, but the re-reading I've been doing reveals an issue with the current table and supporting text.

We're giving one benefits list for 2 Proof Formats (JSON+JWTs, JSON-LD+JWTs) -- and only mentioning JWTs for that list -- and one benefits list for 1 Proof Format (JSON-LD+LD-Proofs).

It seems to me that we should have one text section for each column, and the section title should match the column.  Even if no-one agrees that we should split the current JSON+JWTs and JSON-LD+JWTs into 2 sections, I think that section should be retitled to `Benefits of JSON+JWTs or JSON-LD+JWTs` or (better!) **`Benefits of JSON or JSON-LD with JWTs`** (and the other from `Benefits of JSON-LD and LD-Proofs` to `Benefits of JSON-LD+LD-Proofs` or (better!) **`Benefits of JSON-LD with LD-Proofs`**).

The order of elements might also be better switched (i.e., `JWTs with JSON`, `JWTs with JSON-LD`, `LD-Proofs with JSON-LD`).

It might *also* be worth splitting out the ZKP sub-set of the LD-Proofs column, given that `JWTs with JSON-LD` could arguably be discussed as a sub-set of `JWTs with JSON`.

File: index.html (hunk 4 above)
Comment (brentzundel) [reply index.html]: fixed in [9a0eaac](https://github.com/w3c/vc-imp-guide/pull/28/commits/9a0eaac6b21a3c4c4a8281809a1e809f781aadfc)

File: index.html (hunk 2 above)
Comment (brentzundel) [reply index.html]: fixed in [9a0eaac](https://github.com/w3c/vc-imp-guide/pull/28/commits/9a0eaac6b21a3c4c4a8281809a1e809f781aadfc)

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html]: fixed in [9a0eaac](https://github.com/w3c/vc-imp-guide/pull/28/commits/9a0eaac6b21a3c4c4a8281809a1e809f781aadfc)

File: index.html (hunk 3 above)
Comment (brentzundel) [reply index.html]: fixed in [90a1587](https://github.com/w3c/vc-imp-guide/pull/28/commits/90a1587738291f5c434366809c153285996a1ae4)

**Comment by brentzundel** [comment]
> I will likely have more on this, but the re-reading I've been doing reveals an issue with the current table and supporting text.
> 
> We're giving one benefits list for 2 Proof Formats (JSON+JWTs, JSON-LD+JWTs) -- and only mentioning JWTs for that list -- and one benefits list for 1 Proof Format (JSON-LD+LD-Proofs).
> 
> It seems to me that we should have one text section for each column, and the section title should match the column. Even if no-one agrees that we should split the current JSON+JWTs and JSON-LD+JWTs into 2 sections, I think that section should be retitled to `Benefits of JSON+JWTs or JSON-LD+JWTs` or (better!) **`Benefits of JSON or JSON-LD with JWTs`** (and the other from `Benefits of JSON-LD and LD-Proofs` to `Benefits of JSON-LD+LD-Proofs` or (better!) **`Benefits of JSON-LD with LD-Proofs`**).
> 
> The order of elements might also be better switched (i.e., `JWTs with JSON`, `JWTs with JSON-LD`, `LD-Proofs with JSON-LD`).

Now that each party has added their prose, should we also take a stab at combining the tables?

> It might _also_ be worth splitting out the ZKP sub-set of the LD-Proofs column, given that `JWTs with JSON-LD` could arguably be discussed as a sub-set of `JWTs with JSON`.

This text was moved in PR #32

**Comment by TallTed** [comment]
> Now that each party has added their prose, should we also take a stab at combining the tables?

I think that would be of great benefit to all consumers of this document. I also think I'm not the best to attempt this combination.


> > It might _also_ be worth splitting out the ZKP sub-set of the LD-Proofs column, given that `JWTs with JSON-LD` could arguably be discussed as a sub-set of `JWTs with JSON`.
> 
> This text was moved in PR #32

Not the text, nor the column, [to which I was referring](https://github.com/w3c/vc-imp-guide/blob/5ff637c784e5bec18463221499a55af27f8746e1/index.html#L1002) --
```
The table below compares three syntax and proof format ecosystems; JSON+JWTs,
JSON-LD+JWTs, and JSON-LD+LD-Proofs. Readers should be aware that
Zero-Knowledge Proofs are currently proposed as a sub-type of LD-Proofs and
thus fall into the final column below.
```

**Review by TallTed — APPROVED** [review]
Table consolidation aside (which is moving to another, tentative, PR), this PR is fine from my perspective.

**Pull request merged**